medusa/reconstruct/expand.py

```diff
@@ -529,6 +272,8 @@ def _build_ensemble_from_gapfill_solutions(model,solutions,universal=None):
         members += [member]
 
     ensemble.members = members
+#     save_directory = ("/Users/basazinbelhu/probannopy/iterative_pfba/data/medusa_gapfilled_model_pickle/" +str(ensemble.name)+".pkl")
+#     ensemble.to_pickle(save_directory)
```

Comment:
We shouldn't save the ensemble, just return it. If a user needs to save the ensemble, they can do so using the returned object in their own script.